test/functional/apps/dashboard/group5/dashboard_settings.ts

```diff
@@ -20,6 +21,14 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   describe('dashboard settings', () => {
     let originalTitles: string[] = [];
 
+    const checkDashboardTitle = async (expectedTitle: string) => {
+      expect(await browser.getTitle()).to.equal(`${expectedTitle} - Elastic`);
+      await retry.try(async () => {
+        const breadcrumb = await globalNav.getLastBreadcrumb();
+        expect(breadcrumb).to.equal(`Editing ${expectedTitle}`);
+      });
```

Comment:
We used to be checking for the breadcrumb title like this:

```typescript
await retry.try(async () => {
   expect((await globalNav.getLastBreadcrumb()) === newTitle);
});
```
That is why it didn't need to include the `"Editing"` prefix previously. I've fixed this by switching to `expect(....).to.equal("Editing ....")`

Reply:
Is it possible to test against the doc title here? I might be wrong, but it looks like we're only ensuring that the breadcrumb is correct.

Reply:
Yup - sorry, it's easy to miss, but I'm checking that before I check the breadcrumb:

```typescript
expect(await browser.getTitle()).to.equal(`${expectedTitle} - Elastic`);
```

Reply:
Ah perfect, thank you!